modules/reference/pages/systems/system-details/sd-custom-info.adoc

```diff
@@ -1,15 +1,9 @@
 [[sd-custom-info]]
-= SD Custom Info
+= Custom Info
 
-This subtab provides completely customizable information about the system.
-Unlike [guimenu]``Notes``, [guimenu]``Custom Info`` is structured, formalized, and can be searched.
+The menu:Systems[Details > Custom Info] section lists the custom system information that has been applied to the selected client.
```

Comment:
I think we'd better use ``menu:System Details[Custom Info]``.

Reply:
See other comment.

Reply:
![image](https://user-images.githubusercontent.com/3914967/89362746-fb5fd900-d711-11ea-9d67-24778d1d5781.png)